packages/auth-construct/src/construct.ts

```diff
@@ -893,40 +939,41 @@ export class AmplifyAuth
         output.mfaTypes = JSON.stringify(mfaTypes);
       }
     }
-    const oauthMappings = this.providerSetupResult.oauthMappings;
-    if (oauthMappings[authProvidersList.amazon]) {
-      output.amazonClientId = oauthMappings[authProvidersList.amazon];
+    const oAuthMappings = this.providerSetupResult.oAuthMappings;
+    if (oAuthMappings[authProvidersList.amazon]) {
+      output.amazonClientId = oAuthMappings[authProvidersList.amazon];
     }
-    if (oauthMappings[authProvidersList.facebook]) {
-      output.facebookClientId = oauthMappings[authProvidersList.facebook];
+    if (oAuthMappings[authProvidersList.facebook]) {
+      output.facebookClientId = oAuthMappings[authProvidersList.facebook];
     }
-    if (oauthMappings[authProvidersList.google]) {
-      output.googleClientId = oauthMappings[authProvidersList.google];
+    if (oAuthMappings[authProvidersList.google]) {
+      output.googleClientId = oAuthMappings[authProvidersList.google];
     }
-    if (oauthMappings[authProvidersList.apple]) {
-      output.appleClientId = oauthMappings[authProvidersList.apple];
+    if (oAuthMappings[authProvidersList.apple]) {
+      output.appleClientId = oAuthMappings[authProvidersList.apple];
     }
 
     if (this.providerSetupResult.providersList.length > 0) {
       output.socialProviders = JSON.stringify(
         this.providerSetupResult.providersList
       );
       // if any providers were defined, we must expose the oauth settings to the output
-      if (this.oAuthSettings) {
+      const oAuthSettings = this.providerSetupResult.oAuthSettings;
+      if (oAuthSettings) {
         if (this.domainPrefix) {
           output.oauthDomain = `${this.domainPrefix}.auth.${
             Stack.of(this).region
           }.amazoncognito.com`;
         }
 
         output.oauthScope = JSON.stringify(
-          this.oAuthSettings.scopes?.map((s) => s.scopeName) ?? []
+          oAuthSettings.scopes?.map((s) => s.scopeName) ?? []
         );
-        output.oauthRedirectSignIn = this.oAuthSettings.callbackUrls
-          ? this.oAuthSettings.callbackUrls.join(',')
+        output.oauthRedirectSignIn = oAuthSettings.callbackUrls
```

Comment:
Do we need to change all the cases from `oauth` to `oAuth`?

If we do, could you make a global search?